build/goreleaser/linux/rhel_9_amd64.yml

```diff
@@ -0,0 +1,81 @@
+  # RHEL 9 amd64
+  
+  - id: rhel-9-infrastructure-agent
+    builds:
+      - linux-agent-amd64
+      - linux-ctl-amd64
+      - linux-service-amd64
+    package_name: newrelic-infra
+    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el8.{{ .Arch }}"
```

Comment:
shouldn't this template need to be `"newrelic-infra-{{ .Env.TAG }}-1.el9.{{ .Arch }}"` ?

Reply:
good catch, let me check for 8 number in that file